smart/plugin.go

```diff
@@ -166,6 +177,10 @@ func (sc *SmartCollector) GetMetricTypes(_ plugin.ConfigType) ([]plugin.MetricTy
 
 //GetConfigPolicy returns a ConfigPolicy
 func (p *SmartCollector) GetConfigPolicy() (*cpolicy.ConfigPolicy, error) {
-	c := cpolicy.New()
-	return c, nil
+	cp := cpolicy.New()
+	rule, _ := cpolicy.NewStringRule("proc_path", false, "/proc")
+	node := cpolicy.NewPolicyNode()
+	node.Add(rule)
+	cp.Add([]string{nsVendor, nsClass, PluginName}, node)
+	return cp, nil
```

Comment:
@marcintao Using one environment variable was one of my first idea when I started fixing plugins for use with Docker. This was however strongly discouraged by @marcin-krolik and therefore all other plugins have been uniformized using the exact same code than here